### ISSUE TITLE
NTR: Anti-Ribonucleoprotein antibody positivity

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5802]
**Preferred term label:**
 Anti-Ribonucleoprotein antibody positivity

**Synonyms**
 Anti-RNP-antibodies

**Definition (free text, please give PubMed ID)**
 The presence autoantibodies in the serum that react to proteins (70 Kd, A, C) that are associated with U1 RNA and form U1snRNP.

Comment:  High titers of anti-RNP antibodies are diagnostic of mixed connective tissue disorder (MCTD). Anti-RNP antibodies can also be found in patients with systemic lupus erythematosus (SLE). 
 PMID: 29796907 ; DOI: 10.1007/s00296-018-4059-4
 PMID: 15804705 ; DOI: 10.1080/08916930400022715 

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:536   (Disorder)   Systemic lupus erythematosus

**Your nano-attribution (ORCID)**
lmu: chrapp, @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
added as HP:0033028
